scratchpad/.post-create

```diff
@@ -7,9 +7,11 @@ Start Temporal Server:
 
   {cyan temporal server start-dev}
 
-Install ts-node: https://www.npmjs.com/package/ts-node
+Use Node version 16+:
```

Comment:
Can we change this to a more recent node version?

Reply:
i think they'll all get changed together soon

Reply:
(looks like involves changing the create tool in sdk-typescript too)